metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -102,8 +102,11 @@ def run(
         raise SensitiveError() from e
 
     logger.info("Starting metadata ingestion")
-    pipeline.run()
-    logger.info("Finished metadata ingestion")
+    try:
+        pipeline.run()
+        logger.info("Finished metadata pipeline")
+    except Exception as e:
+        logger.error(f"Caught exception while running metadata ingestion: {e}")
```

Comment:
Let's log the error and rethrow so that the process doesn't exit 0

Reply:
Also, we don't want to capture misconfigurations by the user in the telemetry. So, re-throwing is the right thing to do here.

Reply:
I did not want to lose `pipeline.pretty_print_summary` is the main reason I did not re-throw the exception. I pushed a change for non-zero return code. Let me know if that take care of the concern.